<?xml version="1.0"?>
<launch>


  <node name="pepper_hand_pose" pkg="pepper_hand_pose" type="pepper_hand_pose_node" output="screen">
    <param name="numPoints" value="4" />
    <param name="robotIp" value="$(optenv PEPPER_IP 131.254.10.126)" />
    <param name="targetCalibration" value="True" />

  </node>


</launch>
